dht.go

```diff
@@ -72,8 +78,17 @@ type IpfsDHT struct {
 	autoRefresh           bool
 	rtRefreshQueryTimeout time.Duration
 	rtRefreshPeriod       time.Duration
+	bootstrapCfg          opts.BootstrapConfig
 	triggerRtRefresh      chan chan<- error
 
+	triggerBootstrap chan struct{}
```

Comment:
this doesn't seem to be used anywhere

Reply:
Oh, looks like the remanent of a rebase. Have removed this.